docs/source/using_gridappsd/Query Timeseries.rst

```diff
@@ -0,0 +1,55 @@
+Query Time-series database
+-------------------------
+
+Connect to the running GridAPPS-D container
```

Comment:
Consider linking to where this is available in the documentation so we don't have it all over the place.